src/RadhydroSimulation.hpp

```diff
@@ -507,8 +507,12 @@ template <typename problem_t> void RadhydroSimulation<problem_t>::addStrangSplit
 }
 
 template <typename problem_t>
-void RadhydroSimulation<problem_t>::addStrangSplitSourcesWithBuiltin(amrex::MultiFab &state, int lev, amrex::Real time, amrex::Real dt)
+auto RadhydroSimulation<problem_t>::addStrangSplitSourcesWithBuiltin(amrex::MultiFab &state, int lev, amrex::Real time, amrex::Real dt) -> bool
 {
+
+	// start by assuming chemistry burn is successful.
+	bool burn_success = true;
```

Comment:
This is incorrect. We want to be able to change the value of burn_success if primordial chemistry is on. Why doesn't clang-tidy like it @BenWibking

Reply:
Ifdefs are invisible to clang-tidy, which is why we try to avoid them in general. In this case, just silence the clang-tidy warning by adding '// NOLINT' to the end of this line.

Reply:
Ohh that's what `// NOLINT` is for!!